src/index.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Navbar from './components/Navbar';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <BrowserRouter>
+      <Navbar />
```

Comment:
[OPTIONAL] I think it makes more sense to place the Navbar component inside of the App component as it makes part of the UI from the App. This is a little subjective though. Just a reminder that writing semantic code it's important to keep codebases understandable and organized.